README.md

```diff
@@ -1,2 +1,30 @@
-# p4runtime
-Specification documents for the P4Runtime control-plane API
+# P4Rruntime Specification
```

Comment:
typo s/P4Rruntime/P4Runtime
I found this typo a few times in this PR so I recommend sed'ing it / grep'int it

Reply:
ok, thanks